src/neo/SmartContract/InteropService.Blockchain.cs

```diff
@@ -11,6 +12,8 @@ partial class InteropService
     {
         public static class Blockchain
         {
+            private const uint MaxTraceableBlocks = Transaction.MaxValidUntilBlockIncrement;
```

Comment:
I think that it could be public 
```suggestion
            public const uint MaxTraceableBlocks = Transaction.MaxValidUntilBlockIncrement;
```